frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/DiscoverAction.java

```diff
@@ -56,11 +56,11 @@ public void actionPerformed(AnActionEvent anActionEvent) {
 
     Consumer<IGitMacheteRepositorySnapshot> saveAction = repositorySnapshot -> saveDiscoveredLayout(repositorySnapshot,
         GitVfsUtils.getMacheteFilePath(gitRepository), project, getGraphTable(anActionEvent),
-        getBranchLayoutWriter(anActionEvent));
-    Consumer<IGitMacheteRepositorySnapshot> saveAndEditAction = repositorySnapshot -> {
-      saveAction.accept(repositorySnapshot);
```

Comment:
Oh ok so to check my understanding:
* if `saveAction.accept(...)` completed _quickly_ enough, then `if` in `openMacheteFile` (line 87) went into `true` branch and opened the file
* if `saveAction.accept(...)` completed _slowly_ enough, then `if` in `openMacheteFile` (line 87) went into `false` branch and displayed an error

Amirite?

Reply:
:yes: